src/Controller/EnMarche/CitizenInitiativeManagerContoller.php

```diff
@@ -184,4 +188,43 @@ public function contactMembersAction(Request $request, CitizenInitiative $initia
             'form' => $form->createView(),
         ]);
     }
+
+    /**
+     * @Route("/inscrits/imprimer", name="app_citizen_initiative_print_members")
+     * @Method("POST")
+     */
+    public function printMembersAction(Request $request, CitizenInitiative $initiative): Response
+    {
+        if (!$this->isCsrfTokenValid('event.print_members', $request->request->get('token'))) {
+            throw $this->createAccessDeniedException('Invalid CSRF protection token to print members.');
+        }
+
+        $uuids = json_decode($request->request->get('prints'), true);
+
+        if (!$uuids) {
```

Comment:
```
if (!$uuids = json_decode($request->request->get('prints'), true)) {
....
}
```
no ?

Reply:
I fixed in `EventManagerContoller.php`. Here no need of this change, because this class will be removed in your PR https://github.com/EnMarche/en-marche.fr/pull/2025/files#diff-d87cb18a69cc5e9ea0894b04ddd1b631L1.

Reply:
Please revert changes on this file though.

Reply:
It's needed in order that tests pass.